cli/main.c

```diff
@@ -561,7 +561,7 @@ static int latency(int argc, char **argv)
 	}
 
 	ret = switchtec_lat_setup(cfg.dev, cfg.egress, cfg.ingress, 1);
-	if (ret < 0) {
+	if (ret) {
```

Comment:
switchtec_lat_setup() may return positive value on success. What is the return value you see in this case?

Reply:
It's PMON_ERR_LTCY_SRC_PRT_INV (0x00100004) or PMON_ERR_PRT_INV (0x00100002).

Reply:
This doesn't really fix the issue. switchtec_lat_setup() may return positive value on success in current code.